CSharp/Library/Microsoft.Bot.Builder.Azure/AzureStorageQueueActivityLogger.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.History;
+using Microsoft.Bot.Connector;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Azure
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class AzureStorageQueueActivityLogger : IActivityLogger
+    {
+
+        private readonly JsonSerializerSettings _jsonSerializerSettings;
+        private CloudQueueClient _cloudQueueClient;
+        private CloudQueue _queueClient;
```

Comment:
I'd name this _queue or _cloudQueue to be consistent with line 18, and avoid confusions with the _cloudQueueClient which is the real client/